packages/client/components/TeamPrompt/Recurrence/TimeLeftBadge.tsx

```diff
@@ -25,7 +26,7 @@ export const TimeLeftBadge = (props: Props) => {
       <TeamPromptBadge onMouseEnter={openTooltip} onMouseLeave={closeTooltip} ref={originRef}>
         {fromNow} left
       </TeamPromptBadge>
-      {tooltipPortal(`Ends at ${meetingEndTimeDate.toLocaleString()}`)}
+      {tooltipPortal(`Restarts at ${dayjs(meetingEndTimeDate).format('MMM D, YYYY h:mm A')}`)}
```

Comment:
+1 Since the day immediately follows the preposition, grammatically I believe it's "restarts on" and then ideally have "at" preceding the time.

Reply:
<img width="412" alt="Screenshot 2023-02-10 at 15 33 54" src="https://user-images.githubusercontent.com/1017620/218117819-bae445fc-8907-4482-9a5f-3a2de6fc1740.png">

unfortunately `dayjs` doesn't support custom formats so I had to call `format` twice, but looks much better, thank you